kythe/go/extractors/config/extractor.go

```diff
@@ -43,6 +43,14 @@ type Repo struct {
 	ConfigPath string
 }
 
+// GitCopier creates a Repo.Copier that fetches repos via git commandline.
```

Comment:
Ok yea that makes a lot of sense.

For pending follow-up changes this will make even more sense, since we might have more persistence & re-use between calls, so passing context would allow finer grained control on e.g. timeout.